python/lsst/ctrl/bps/wms/htcondor/lssthtc.py

```diff
@@ -207,7 +212,12 @@ def htc_write_attribs(stream, attrs):
         HTCondor job attributes (dictionary of attribute key, value)
     """
     for key, value in attrs.items():
-        print(f'+{key} = "{htc_escape(value)}"', file=stream)
+        if isinstance(value, str):
+            pval = f'"{htc_escape(value)}"'
+        else:
+            pval = value
+
+        print(f'+{key} = {pval}', file=stream)
```

Comment:
Could you please add a brief comment here hinting why this check in necessary here? Since it is so "close" to identical check in `htc_escape()` it took me longer that I'd like to admit to figure out that they serve two different purposes.